src/Symfony/Component/Serializer/Normalizer/ArrayDenormalizer.php

```diff
@@ -64,10 +64,12 @@ public function denormalize($data, $class, $format = null, array $context = arra
     /**
      * {@inheritdoc}
      */
-    public function supportsDenormalization($data, $type, $format = null)
+    public function supportsDenormalization($data, $type, $format = null/*, array $context = array()*/)
     {
+        $context = func_num_args() > 3 ? func_get_arg(3) : array();
```

Comment:
Same as before, adding a new argument to a public method isn't allowed by our BC policy.